From: Luciano Bello <luciano@debian.org>
Subject: nltk is not part of debian yet (check #279422). The wordnet plugin was disable.

--- a/core/controllers/misc/dependencyCheck.py
+++ b/core/controllers/misc/dependencyCheck.py
@@ -74,8 +74,8 @@
     try:
         import nltk
     except Exception, e:
-        packages.append('nltk')
-        packages_debian.append('python-nltk')
+        #packages.append('nltk')
+        #packages_debian.append('python-nltk')
         #TODO
         #packages_mac_port.append()
         msg  = '    If you can not install nltk, please try the following:\n'
@@ -88,8 +88,8 @@
         msg += '        tar -xzvf nltk-2.0b9.tar.gz\n'
         msg += '        cd nltk-2.0b9\n'
         msg += '        python setup.py install'
-        additional_information.append(msg)
-        reasonForExit = True
+        #additional_information.append(msg)
+        #reasonForExit = True
 
     try:
         import extlib.SOAPpy.SOAPpy as SOAPpy
